docs/HOWTO_INTEGRATE_WITH_RENDER.md

```diff
@@ -0,0 +1,152 @@
+    - Click `Authorize Render`
+    - If you have multiple organizations, pick the organization where your forked repository exists.
+    - Decide if you want Render to be installed for all your repositories or only select repositories.
+    - I've chosen only select repositories.
+    - Click install
+    - Authorize your account
+    - It sometimes returns back to the original screen with no instructions on what to do next.  Don't be alarmed! That might mean it worked! Please click `Create New Webservice` again, and you should see your repository linked with a purple button reading `connect`
+- Now you should see a message that is suggesting `Docker`
+  - If not, please enter the following:
+    - `Name` - your name, it helps if it's your organization or campaign name with spoke next to it.
+    - `Environment` - Docker
+    - `Region` - default value
+    - `Branch` : pick the branch from your repository, typically in production, the branchname is main.
+
+## Setting Up Required Environment Variables
+- Under `Advanced`
+  - `Environment Variable`
+    EXAMPLE:  `THE_ENVIRONMENT_VARIABLE` = IT'S VALUE
+    Sometimes, there are additional steps for finding the correct value for your environment variables.  Below is the best grouping of all required environment variables to make your integration successful!
+
+### Auth0 Environment Variables & Instructions
+**Follow [these instructions](https://moveonorg.github.io/Spoke/#/HOWTO_HEROKU_DEPLOY?id=instructions-for-auth0-configuration-variable-setup) to get the correct values for the following environment variables:**
+  - `AUTH0_CLIENT_ID` = [your auth0 client id]
+  - `AUTH0_CLIENT_SECRET` = [your auth0 client secret]
+  - `AUTH0_DOMAIN` = [your auth0 domain: XXXXX.us.auth0.com]
+  - If you're having a hard time with your Auth0 Integration, double check your permissions within your Auth0 Account.
+    - Allowed Callback URLs - https://yourspoke.onrender.com/login-callback, https://yourspoke.onrender.com/login
+    - Allowed Web Origins - https://yourspoke.onrender.com
+    - Allowed Logout URLs - https://yourspoke.onrender.com/logout-callback
+    - Allowed Origins (CORS) - https://yourspoke.onrender.com
+    - In advanced settings turn off the OIDC Conformant.
+### Database values
+**Once you have created your database (instructions below) you will be able to access this information.  Once a database has been created in Render, under Connections, find the following values for:**
+  - `DB_HOST` = [in Render as HostName]
+  - `DB_NAME` = [in Render as Database]
+  - `DB_PASSWORD` = [in Render as Password]
+  - `DB_USER` = [in Render as Username]
+  - `DB_PORT` = [in Render as Port] typically 5432
+
+### Required Environment Variables and exact values  
+**add the following required environment variables and their respective values:**
+  - `DB_TYPE` = pg
+  - `ACTION_HANDLERS` = ""
```

Comment:
I think we figured out that environment variables with empty quotes as the value caused errors, and should be omitted, right?